src/components/app/App.js

```diff
@@ -38,29 +37,22 @@ function App() {
         {
           postsList.map((item, index) => {
             const odd = index % 2 !== 0;
-            // todo: найти в массиве usersList пользователя, айди которого равно user_id в посте (т.е. в данном случае item)
-            //  для этого можно использовать функцию массива find или findIndex
-            //  передать имя и фамилию пользователя как пропсу author в Card
-            //  использовать для этого стринговый литерал ``
-            //  в Card под телом поста срендерить имя автора, используя blockquote-footer класс из бутстрапа
-            //  пример тут: https://hackerthemes.com/bootstrap-cheatsheet/#blockquote-footer
-
 
-            // todo: найти в массиве allComments комментарии, post_id которых = id поста  (т.е. в данном случае item.id)
-            //  для этого можно использовать метод массива filter
-            //  передать этот массив в Card как пропсу под названиес comments
+            const user = usersList.find(user=> user.id === item.user_id);
+            const {first_name, last_name} = user;
+            const author = `${first_name} ${last_name}`;
+            const comments = allComments.filter(value => value.post_id === item.id);
 
-              return <Card post={item} key={item.id} hasImage={odd} />
+            return <PostCard post={item} key={item.id} hasImage={odd} author={author} comments={comments}/>
           })
         }
       </div>
 
       <div className="d-flex posts-container">
-      {/*  todo: срендерить тут список пользователей, используя компонент UserCard */}
-
+          {usersList.map((item, index) => {return(<UserCard key={item.id} user={item}/>)})}
```

Comment:
if you want to write in one line you can simplify event more arrow function 
`usersList.map((item, index) => (<UserCard key={item.id} user={item}/>))`